src/main/java/org/kohsuke/groovy/sandbox/SandboxTransformer.java

```diff
@@ -322,8 +323,14 @@ private Expression innerTransform(Expression exp) {
                 try (StackVariableSet scope = new StackVariableSet(this)) {
                     Parameter[] parameters = ce.getParameters();
                     if (parameters != null) {
```

Comment:
Yup yup. No idea whether it was by design or not, but it does appear to be the right behavior.